ua-snap/iem-webapp
# Design final push

Closes #566 
Closes #547 
Closes #556 
Closes #567 
Closes #568 
Closes #553 
Closes #551 
Closes #425 

Testing:

- Check ticket #556 for layout changes overall.  This moves a lot of text blocks around (into a columnar format) + implements the HeaderBlock.  Please test + fix for mobile / print.  The main pieces to focus on are the HeaderBlock, though.  #547 is basically a dupe here.
- See if you can verify the OpenGraph Preview image is correctly changed/fixed. (#567)
- Check under Temperature Indicators table + Precip Indicators table.  For both, there should just be a single link to the Data page (#568)
- Check the AA/EO footer phrasing (#566)
- Check the footer v. #553 -- I made a few small text adjustments (added a "Contact us" thing) + removed the `<br>`s because it was wreaking havoc on the layout.
- About Page / Glossary (#551)  (also closes #425)
